packages/core/src/generator/index.ts

```diff
@@ -645,7 +646,8 @@ function applyScope(css: string, scope?: string) {
 export function movePseudoElementsEnd(selector: string) {
   const pseudoElements = selector.match(/::[\w-]+(\([\w-]+\))?/g)
   if (pseudoElements) {
-    pseudoElements.forEach(e => (selector = selector.replace(e, '')))
+    for (const e of pseudoElements)
```

Comment:
Did you use some formatting tools? (It's not related to the question. I prefer es6 syntax 😅)